packages/components/src/components/ScalarCard/ScalarCard.vue

```diff
@@ -46,13 +46,14 @@ defineOptions({ inheritAttrs: false })
 const { cx } = useBindCx()
 </script>
 <template>
-  <section
```

Comment:
This was my bad making all cards a `region`, in hindsight that's a bit overkill. Group makes more sense.